ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -112,24 +76,66 @@ struct SlideOverPanel<Content: View>: View {
         }
     }
 
-    private func percentTransistionOffset() -> CGFloat {
-        // override offset if required
+    private var percentTransistionOffset: CGFloat {
+        // Override offset if required
         if let percentDriveTransition = percentTransition {
             // parent view wants to take over primarily to conform user drag gesture
             if percentDriveTransition >= 0 && percentDriveTransition <= 1 {
-                return expandedContentOffset() + collapsedContentOffset() * CGFloat(1 - percentDriveTransition)
+                return expandedContentOffset + collapsedContentOffset * CGFloat(1 - percentDriveTransition)
             }
         }
         return CGFloat.zero
     }
 
-    private func expandedContentOffset() -> CGFloat {
+    private var expandedContentOffset: CGFloat {
         return CGSize.zero.width
     }
 
-    private func collapsedContentOffset() -> CGFloat {
+    private var collapsedContentOffset: CGFloat {
         return slideOutPanelWidth * contentWidthSizeRatio
     }
+
+    public var body: some View {
```

Comment:
The file content order is public -> internal -> private according to the [swift-guide](https://github.com/microsoft/swift-guide/blob/gh-pages/FileContentOrdering.md).

Reply:
marked as internal to conform to `View` protocol